src/css/main.scss

```diff
@@ -41,6 +41,6 @@ iframe {
 }
 
 i {
-  color: $icon-color;
+  //color: $icon-color;
```

Comment:
rm unused comment or
check icon colors in styleguide and maybe add new color variable